src/main/webapp/index.html

```diff
@@ -3,17 +3,18 @@
 
 <title>ZoomTube</title>
 <link rel="stylesheet" href="style.css" />
-<script src="script.js"></script>
 <form action="/lecture" method="POST">
-  <h2>Add a new lecture!<h2>
+  <h2>Add a new lecture!</h2>
   <h3>Enter a name for this lecture:</h3>
-  <input type="text" placeholder="Lecture Name" name="name-input">
-  <br/>
-  <h3>Enter the YouTube link for this lecture:<h3>
-  <input type="text" placeholder="Name" name="video-input">
-  <br/>
-  <br/>
+  <input type="text" placeholder="Lecture Name" name="name-input" />
+  <br />
+  <h3>Enter the YouTube link for this lecture:</h3>
+  <input type="text" placeholder="Name" name="video-input" />
+  <br />
+  <br />
   <div id="submit-button">
-    <input type="submit"/>
+    <input type="submit" />
   </div>
 </form>
+<ul id="lecture-list"></ul>
+<script src="script.js"></script>
```

Comment:
An alternative to bringing script to the end of the doc is adding the `defer` tag, which waits until the entire index.html is ready before running.  IDK which is preferable
```suggestion
<script src="script.js" defer></script>
```

Reply:
fixed